### PR TITLE
No build file source

### PR DESCRIPTION
**Pull request by jcrugzz**
Be able to handle not having a build-file source and still support asset routes

### CODE REVIEW

File: lib/routes/assets.js [hunk 1]
```diff
@@ -35,22 +40,25 @@ module.exports = function (app) {
...
         }));
       }
 
-      if (gzip) {
-        res.setHeader('Content-Encoding', 'gzip');
-      }
-
-      res.setHeader('Content-Length', build.source.length);
-      res.send(build.source);
+      var url = extension === '.map' ? build.url + '.map' : build.url;
```
Comment (bmeck) [inline lib/routes/assets.js]: is there ever a case where .map files exist on their own?

**Comment by Swaagie** [comment]
LGTM, gzip was a temporary fix iirc before we integrated with the CDN

File: lib/routes/assets.js (hunk 1 above)
Comment (jcrugzz) [inline lib/routes/assets.js]: not in how we publish currently, it always lives in the same `fingerprint` bucket as the source for them to work properly in browsers.

**Comment by bmeck** [comment]
lgtm

**Pull request merged**